<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="portGroupBox">
     <property name="title">
      <string>Select serial port</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="portCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="paramGroupBox">
     <property name="title">
      <string>Select parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="baudRateLabel">
        <property name="text">
         <string>Baudrate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baudRateCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="dataBitsLabel">
        <property name="text">
         <string>Data bits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="dataBitsCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="parityLabel">
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="parityCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="stopBitsLabel">
        <property name="text">
         <string>Stop bits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stopBitsCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="flowLabel">
        <property name="text">
         <string>Flow control</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="flowCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
